spec/models/human_spec.rb

```diff
@@ -6,4 +6,45 @@
       expect(build(:human, roles: [student])).to be_valid
     end
   end
+  describe '#visible_courses' do
```

Comment:
In general these tests are great but I don't love the mixing of the role and user terminology; it makes it confusing what methods belong to what classes:

- `visible_courses` belongs to the human class and so should be called on a `Human` instance, not on a `Role` instance (student, admin, ta)

Reply:
Agreed - I've made sure we're only calling it on `Human`s